recipes/defusedxml/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{%set name = "defusedxml" %}
+{%set version = "0.4.1" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "cd551d5a518b745407635bb85116eb813818ecaf182e773c35b36239fc3f2478" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: {{ build_num }}
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - defusedxml
+
+about:
+  home: https://bitbucket.org/tiran/defusedxml
+  license: PSFL 2.0
+  license_file: LICENSE
+  license_family: Proprietary
```

Comment:
Going by [`conda-build/metadata.py`](https://github.com/conda/conda-build/blob/master/conda_build/metadata.py), it's `PSF`